python/ray/train/constants.py

```diff
@@ -55,6 +57,13 @@ def _get_defaults_results_dir() -> str:
 # Deprecated configs can use this value to detect if the user has set it.
 _DEPRECATED_VALUE = "DEPRECATED"
 
+# Map of supported accelerators to the environment variable that
+# are available in worker metadata as resource_ids.
+SUPPORTED_ACCELERATOR_DEVICES_TO_ENV_VAR = {
+    ray_constants.NEURON_CORES: ray_constants.NEURON_RT_VISIBLE_CORES_ENV_VAR,
+    ray_constants.TPU: ray_constants.TPU_VISIBLE_CHIPS_ENV_VAR,
```

Comment:
I realize I said that we should do this in a previous comment, but I'm honestly not familiar how TPU sharing logic works. Maybe we can leave this one out for now?

Reply:
Removed TPU for now